contracts/interfaces/IFacadeWrite.sol

```diff
@@ -84,16 +84,13 @@ interface IFacadeWrite {
     /// @param rToken The address of the RToken
     /// @param governance The address of the new governance
     /// @param timelock The address of the timelock
-    event GovernanceCreated(
-        IRToken indexed rToken,
-        address indexed governance,
-        address indexed timelock
-    );
+    event GovernanceCreated(IRToken rToken, address governance, address timelock);
```

Comment:
I think we should keep all 3 indexed here, since this only runs once per RToken instance.